consensus/tests/integration_tests.rs

```diff
@@ -714,6 +714,18 @@ async fn goref_tx_small_concurrent_test() {
     json_concurrency_test("tests/testdata/goref-905-tx-265-blocks.json.gz").await
 }
 
+#[ignore]
+#[tokio::test]
+async fn goref_tx_big_test() {
+    json_test("tests/testdata/goref-1.6M-tx-10K-blocks.json.gz").await
```

Comment:
The test can't reference a file outside this repo. This should be deleted or rewritten in a way that somehow fetches this file